_data/simple-icons.json

```
@@ -6160,6 +6160,11 @@
             "hex": "CC092F",
             "source": "https://www.iceland.co.uk"
         },
+        {
+            "title": "ICICI Bank",
+            "hex": "AE282E",
+            "source": "https://commons.wikimedia.org/wiki/File:ICICI_Bank_Logo.svg"
```

Can we use this instead?
```suggestion
            "source": "https://www.icicibank.com/ms/aboutus/annual-reports/2022-23/icici/assets/images/home-page/logo.svg"
```

---

@adamrusted Please take a look. We can use source SVG from the official website itself

---

@PratyushJayachandran - if it's on the homepage, we should instead use the homepage URL.

---

It's actually not available as svg on the homepage that's why I put this link.

---

@PratyushJayachandran, can you update the URL to the page you found the SVG on?

---

> @PratyushJayachandran, can you update the URL to the page you found the SVG on?

Done